src/utils/helpers.coffee

```diff
@@ -64,6 +65,15 @@ exports.excludeFile = (fileName, options) ->
             # Only instrument files that are inside the project.
             excluded = true
 
+        # For each exclude value try to use it as a pattern to exclude files
+        exclude.map (pattern) ->
+            glob.sync pattern,
```

Comment:
The one thing that's bad about this is that we'll end up re-running `glob.sync` for the same variables over and over again.  But, without rewriting half this project, I guess this works.  :P  I'll merge this, and see if I can come up with a clever way to make this a little better.  We could cache the result of glob.sync() for each value in options.exclude...